.cirrus.yml

```diff
@@ -212,40 +212,3 @@ task:
         cd $ENGINE_PATH/src
         ./flutter/tools/fuchsia/build_fuchsia_artifacts.py --engine-version HEAD --runtime-mode debug --no-lto --archs x64
         cd $ENGINE_PATH/src/flutter
-
-# WINDOWS
-task:
-  gce_instance:
-    image_project: flutter-cirrus
-    image_name: flutter-engine-windows-server-2016
-    zone: us-central1-a
-    platform: windows
-    cpu: 32
-    memory: 32Gb
-    disk: 50
-  env:
-    # Cirrus is somehow not picking up the environment variables set in the VM image.
-    PATH: "c:/depot_tools;c:/MinGit/cmd;$PATH"
-    DEPOT_TOOLS_WIN_TOOLCHAIN: 0
-    GYP_MSVS_OVERRIDE_PATH: "c:/Program Files (x86)/Microsoft Visual Studio/2017/Community"
-    ENGINE_PATH: "c:/flutter/engine"
-  setup_script: |
-    REM robocopy can return 1 for successful copy; suppress its error code.
-    REM move somehow doesn't work as it complains that the file is being used by another process.
-    robocopy %CIRRUS_WORKING_DIR% %ENGINE_PATH%/src/flutter /MIR| (cmd /s /c exit /b 0)
-    cd %ENGINE_PATH%/src
-    gclient sync
-  matrix:
-    - name: build_and_test_windows_unopt_debug
-      compile_host_script: |
-        cd %ENGINE_PATH%/src
-        python flutter/tools/gn --runtime-mode debug --unoptimized
-        ninja -C out/host_debug_unopt
-      test_host_script: |
-        cd %ENGINE_PATH%/src
-        python flutter/testing/run_tests.py --type=engine
```

Comment:
The LUCI job "Windows Host Engine" only builds `host_debug`, not `host_debug_unopt`. We can safely remove the `build_windows_opt_debug` Cirrus task now, but it would be better to remove the Cirrus task `build_and_test_windows_unopt_debug` when we added `host_debug_unopt` build to the LUCI recipe. CC @dnfield

Reply:
On LUCI we are running tests on host_debug, do we really need to keep the host_debug_unopt? worst case scenario we can build host_debug_unop in the same recipe and it will only add a couple of minutes to the builder execution time.

Reply:
I think the `opt` build to skip some `FML_DCHECK` so it's nice to run tests with `unopt`.

Reply:
Let's land this first to alleviate our test pressure. https://github.com/flutter/flutter/issues/54464 created to track the `unopt` build and test in LUCI.